src/release/process.md

```diff
@@ -106,7 +106,12 @@ Send a PR to the master branch to:
 
 Decide on a time to do the release, T.
 
-- **T-30m** - Run the following command in a shell with [AWS
+- **T-50m** - Ensure there is someone with access to the `@rustlang` Twitter
+  account who'll be able to tweet as soon as we publish the blog post. [Steve
```

Comment:
I think this is too high a bar. I think sending a ping makes sense, but I don't think that we should block on the response. The twitter post is not a critical part of the release, IMO.

Reply:
Both of us doing releases can log into the Twitter account, so even if we can't reach out who usually tweets stuff from the account we should be able to post a tweet on our own.

I personally consider tweets to be the way we publicly announce the release is out, along with the blog post, so I'd say we should make sure we have a tweet out.

Reply:
Hm, I guess that's true in some sense, but I also don't see it as critical that we *do* announce a new release in a push fashion. It makes sense for the release blog post to go out, but the tweet being "late" doesn't seem like a problem to me.

Reply:
Do you have examples in mind of cases where we wouldn't be able to post the tweet? I can't personally think of any, but I'm probably missing something :)

Reply:
I personally don't really post on Twitter at all, and while I could learn to do so, I think posting the tweet is not necessary for a release to happen; it adds additional complexity.

Part of this is that in the future I'd like to get us to a point where the release is even more hands off than today, and while Twitter does have an API etc, I'm much more hesitant about letting bots post to our twitter than tagging commits and such.

Reply:
I think if we get to a point where posting the tweet is one of the only manual steps we can reconsider, but for now I'd prefer if we keep tweeting as part of our regular checklist.

I could see us automating tweeting as part of merging blog posts in the future, so the automation doesn't really have to be specific to the release process. Personally I think we're still far from that.

Reply:
So I'll be honest, I don't want to learn to push the right buttons and login to 1password, then twitter, etc. for each release I run; I don't consider the tweet anywhere near high priority enough for that to be necessary to execute on the release. I think it should happen, yes, but if it happens +0 or +N hours from the actual release seems unimportant to me... I'm trying to understand why you seem to think differently, but so far don't think I've heard concrete arguments. It feels clear to me that announcing the release and actually running it need not be coupled.